docs/migration_guide.md

```diff
@@ -42,28 +34,25 @@ you follow the steps below.
 3. Get the latest Harbor release package from Github:
    https://github.com/goharbor/harbor/releases
 
-4. Before upgrading Harbor, perform migration first.  The migration tool is delivered as a docker image, so you should pull the image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
+4. Before upgrading Harbor, perform migration first.  The migration tool is delivered as a docker image, so you should pull the image from docker hub. Replace [tag] with the release version of Harbor (for example, v1.9.0) in the command below:
     ```
     docker pull goharbor/harbor-migrator:[tag]
     ```
 
-5. Upgrade from `harbor.cfg` to `harbor.yml`
+5. If you are upgrading from v1.7.x, migrate from `harbor.cfg` to `harbor.yml`.
     **NOTE:** You can find the ${harbor_yml} in the extracted installer you got in step `3`, after the migration the file `harbor.yml` 
     in that path will be updated with the values from ${harbor_cfg}
     
     ```
     docker run -it --rm -v ${harbor_cfg}:/harbor-migration/harbor-cfg/harbor.yml -v ${harbor_yml}:/harbor-migration/harbor-cfg-out/harbor.yml goharbor/harbor-migrator:[tag] --cfg up
     ```
-    **NOTE:** The schema upgrade and data migration of Database is performed by core when Harbor starts, if the migration fails,
-    please check the log of core to debug.
-
-6. Under the directory `./harbor`, run the `./install.sh` script to install the new Harbor instance. If you choose to install Harbor with components like Notary, Clair, and chartmuseum, refer to [Installation & Configuration Guide](../docs/installation_guide.md) for more information.
+    **NOTE:** The schema upgrade and data migration of the database is performed by core when Harbor starts, if the migration fails, please check the log of core to debug.
 
+6. Under the directory `./harbor`, run the `./install.sh` script to install the new Harbor instance. If you choose to install Harbor with components such as Notary, Clair, and chartmuseum, refer to [Installation & Configuration Guide](../docs/installation_guide.md) for more information.
 
-### Roll back from an upgrade
-For any reason, if you want to roll back to the previous version of Harbor, follow the below steps:
 
-**NOTE:** Roll back doesn't support upgrade across v1.5.0, like from v1.2.0 to v1.7.0. This is because Harbor changes DB to PostgreSQL from v1.7.0, the migrator cannot roll back data to MariaDB.    
+## Roll Back from an Upgrade
+If, for any reason, you want to roll back to the previous version of Harbor, follow the below steps: 
```

Comment:
maybe communicate that we can rollback during upgrade to the state before upgrade started, but harbor doesn't support downgrades in general